pkg/registry/garden/seed/strategy.go

```diff
@@ -40,6 +42,13 @@ func (seedStrategy) NamespaceScoped() bool {
 func (seedStrategy) PrepareForCreate(ctx genericapirequest.Context, obj runtime.Object) {
 	seed := obj.(*garden.Seed)
 	seed.Status = garden.SeedStatus{}
+
+	finalizers := sets.NewString()
+	if !finalizers.Has(gardenv1beta1.GardenerName) {
+		finalizers.Insert(gardenv1beta1.GardenerName)
+	}
+	seed.Finalizers = finalizers.UnsortedList()
```

Comment:
Can you please do the same for the finalizer modifications in `pkg/registry/garden/shoot/strategy.go`?

Reply:
done